### PR TITLE
Added waypoints on map

### PR DESCRIPTION
**Pull request by matiasbavera**
## What's new
The end-user can now see the waypoints of the map.

image

image

image

### CODE REVIEW

**Comment by matiasbavera** [comment]
@ChawinTan  and @vallq What do you think about the color and the word style?

**Comment by ChawinTan** [comment]
I think the color of the points is good. I like the word style in that it emphasizes itself really well and is easily distinguishable but is slightly hard to read when I am using my laptop (a smaller screen).

**Comment by matiasbavera** [comment]
> I think the color of the points is good. I like the word style in that it emphasizes itself really well and is easily distinguishable but is slightly hard to read when I am using my laptop (a smaller screen).

Thanks for the feedback!

**Comment by matiasbavera** [comment]
Rebased.

**Comment by koonpeng** [comment]
how about different shapes for the waypoints? To make it more distinct from the robots.

File: src/components/schedule-visualizer/tests/waypoints-overlay.test.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import L from 'leaflet';
+import { Map as LMap } from 'react-leaflet';
+import Waypoint from '../waypoint';
+import WaypointsOverlay from '../waypoints-overlay';
+import getBuildingMap from '../../../mock/data/building-map';
+
+test('Render robots correctly', async () => {
```
Comment (koonpeng) [inline src/components/schedule-visualizer/tests/waypoints-overlay.test.tsx:9]: typo here?

Comment (matiasbavera) [reply src/components/schedule-visualizer/tests/waypoints-overlay.test.tsx:9]: You're right! Thank you.

**Comment by matiasbavera** [comment]
> how about different shapes for the waypoints? To make it more distinct from the robots.

How about squares? 
image

**Review by koonpeng — COMMENTED** [review]
image
I think we should make the waypoint layers on top of the door / lifts.

image
Do you think the waypoints should be on top of the robots?

The waypoints don't have any actions on click so let's remove the click cursor.

**Comment by matiasbavera** [comment]
> I think we should make the waypoint layers on top of the door / lifts.

Yeah, sounds good.

> The waypoints don't have any actions on click so let's remove the click cursor.

You're right, I'll remove that.

> Do you think the waypoints should be on top of the robots?

Yeah, it looks better.

**Review by koonpeng — APPROVED** [review]
LGTM, I'm not sure if it's better for the waypoints to on on top or below the robots, but let's just go with on top for now, we can change it easily if more people prefer below.

**Pull request merged**